README.md

```diff
@@ -99,7 +99,8 @@ If xiFDR is very slow or crashing, restart the program by increasing the allocat
 
 
 #### prefilters
-xiSEARCH provides many features of CSMs that may be used to prefilter the results prior to FDR estimation. Doing so equally on targets and decoys prior to FDR estimation retains the accuracy of FDR, while doing score filtering or other filters post FDR estimation generates results with unknown error rates. We recommend doing this only after having a look at the FDR-filtered results without any prefilters.
+xiSEARCH provides many features of CSMs that may be used to prefilter the results prior to FDR estimation. Doing so equally on targets and decoys prior to FDR estimation retains the accuracy of FDR, while doing score filtering or other filters post FDR estimation generates results with unknown error rates. We recommend doing this only after having a look at the FDR-filtered results without any prefilters. 
+If running without prefilters already generates warnings about not enough TTs at the level of interest (links or protein pair) when running FDR calculations without any prefilters, prefilters should NOT be used.
```

Comment:
I don't think this should be quite as strict - there are probably cases where pre-filter actually filter out false positives selectively enough that a previously unreliable FDR becomes reliable.
```suggestion
If running without pre-filters already generates warnings about not enough TTs at the level of interest (links or protein pair), pre-filters are often not making the FDR more reliable.
```